Reaktoro/Optimization/Regularizer.cpp

```diff
@@ -537,6 +537,10 @@ auto Regularizer::Impl::regularize(Matrix& dgdp, Matrix& dbdp) -> void
 
 auto Regularizer::Impl::recover(OptimumState& state) -> void
 {
+    // Skip if echelonization should not be performed or A(echelon) was not computed
+    if(!params.echelonize || !A_echelon.size())
+        return;
+
```

Comment:
These changes here are not really related to the rest of the PR. But they are small enough to consider here! ;)